src/plugins/FileService/arweave/index.ts

```diff
@@ -0,0 +1,101 @@
+import { Attachment } from '@dimensiondev/common-protocols'
+import Arweave from 'arweave/web'
+import type Transaction from 'arweave/web/lib/transaction'
+import { landing } from '../constants'
+import { sign } from './remote-signing'
+import token from './token.json'
+import { isNil, isEmpty } from 'lodash-es'
+
+const stage: Record<Transaction['id'], Transaction> = {}
+
+const instance = Arweave.init({
+    host: 'arweave.net',
+    port: 443,
+    protocol: 'https',
+})
+
+export function makeFileKey(length = 16) {
+    let key = ''
+    const table = 'ABDEFGHJKMNPQRTWXYadefhijkmnprstuvwxyz03478'
```

Comment:
Where does this table come from?

Reply:
`A - Z` + `a - z` + `0 - 9` then remove similar character before